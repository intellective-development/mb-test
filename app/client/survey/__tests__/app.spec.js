import * as React from 'react';
import { shallow } from 'enzyme';
import App from '../app';

const survey = {id: 35, token: '-s2b0re4Vq12_umlmdkh0WiMEessPztm7oiOyruvWzc', score: null, comment: null, user_id: 1, order_id: 38, state: 'pending', created_at: '2017-03-02 14:46:07', updated_at: '2017-03-02 14:46:07'};
const referral_code = 'abc123';

it('renders', () => {
  const component = shallow(<App survey={survey} referralCode={referral_code} />);
  expect(component).toMatchSnapshot();
});
