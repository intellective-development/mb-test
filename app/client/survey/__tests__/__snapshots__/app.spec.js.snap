// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<OrderSurveyFormFields
  submitForm={[Function]}
  survey={
    Object {
      "comment": null,
      "created_at": "2017-03-02 14:46:07",
      "id": 35,
      "order_id": 38,
      "score": null,
      "state": "pending",
      "token": "-s2b0re4Vq12_umlmdkh0WiMEessPztm7oiOyruvWzc",
      "updated_at": "2017-03-02 14:46:07",
      "user_id": 1,
    }
  }
/>
`;
